app/renderer/components/preferences/payment/addFundsDialog/steps/addFundsWizardAddress.js

```diff
@@ -0,0 +1,201 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Components
+const React = require('react')
+const {GroupedFormTextbox} = require('../../../../common/textbox')
+const ClipboardButton = require('../../../../common/clipboardButton')
+
+// Actions
+const appActions = require('../../../../../../../js/actions/appActions')
+
+// Styles
+const {StyleSheet, css} = require('aphrodite')
+const globalStyles = require('../../../../styles/global')
+const {addFundsDialogMinHeight} = require('../../../../styles/global').spacing
+const ethIcon = require('../../../../../../extensions/brave/img/ledger/cryptoIcons/ETH_icon.svg')
+const btcIcon = require('../../../../../../extensions/brave/img/ledger/cryptoIcons/BTC_icon.svg')
+const ltcIcon = require('../../../../../../extensions/brave/img/ledger/cryptoIcons/LTC_icon.svg')
+const batIcon = require('../../../../../../extensions/brave/img/ledger/cryptoIcons/BAT_icon.svg')
+
+class AddFundsWizardAddress extends React.Component {
+  constructor (props) {
+    super(props)
+    this.onCopy = this.onCopy.bind(this)
+  }
+
+  get currency () {
+    return this.props.currency
+  }
+
+  onCopy () {
+    if (!this.addressInputNode) {
+      return
+    }
+    appActions.clipboardTextCopied(this.addressInputNode.value)
+  }
+
+  get copyToClipboardButton () {
+    return (
+      <ClipboardButton
+        bottomTooltip
+        className={globalStyles.appIcons.clipboard}
+        copyAction={this.onCopy}
+      />
+    )
+  }
+
+  render () {
+    return (
+      <div
+        className={css(
+          styles.addFundsWizardAddress,
+          this.currency === 'eth' && styles.addFundsWizardAddress_eth,
+          this.currency === 'btc' && styles.addFundsWizardAddress_btc,
+          this.currency === 'ltc' && styles.addFundsWizardAddress_ltc,
+          this.currency === 'bat' && styles.addFundsWizardAddress_bat
+      )}>
+        <header data-l10n-id='addFundsWizardAddressHeader' />
+        <div className={css(styles.addFundsWizardAddress__main)}>
+          <main className={css(styles.addFundsWizardAddress__inputBox)}>
+            <GroupedFormTextbox type='text'
```

Comment:
Is this not be applied here? https://github.com/brave/browser-laptop/pull/7164#discussion_r100586892

Reply:
I don't think so. works normally in this case

Reply:
thanks for checking :-)